packages/@sfx/search-plugin/test/unit/common/search-plugin.test.ts

```diff
@@ -0,0 +1,60 @@
+import { expect, sinon, stub } from '../../utils';
+import SearchPlugin from '../../../src/search-plugin';
+import SearchPluginOptions from '../../../src/search-plugin';
+import { BridgeConfig, Query } from 'groupby-api';
+import * as SearchPackage from 'groupby-api';
+
+describe('SearchPlugin', () => {
+  let searchPlugin: any;
+
+  beforeEach(() => {
+    let options = {
+      customerId: 'testClientId',
+      https: true,
+    };
+
+    searchPlugin = new SearchPlugin(options);
+  });
+
+  describe('metadata getter', () => {
+    it('should have the name `search`', () => {
+      expect(searchPlugin.metadata.name).to.equal('search');
+    });
+
+    it('should not specify any dependencies', () => {
+      expect(searchPlugin.metadata.depends).to.deep.equal([]);
+    });
+  });
+
+  describe('constructor()', () => {
+    it('should create a new instance of Search BrowserBridge with options', () => {
+      const browserBridgeInstance = { a: 'a' };
+      const SearchBrowserBridge = stub(SearchPackage, 'BrowserBridge').returns(browserBridgeInstance);
+      const config: BridgeConfig = {
+        timeout: 2000,
+      };
+
+      let options = {
```

Comment:
This looks like it's also part of the Arrange phase of testing, so you can remove the above newline.